PythonScript/src/MenuManager.cpp

```diff
@@ -956,7 +959,7 @@ idx_t MenuManager::findMenuCommand(HMENU hParentMenu, const TCHAR *parentMenuNam
 			tstring thisMenuName = formatMenuName(strBuffer);
 			if (NULL == menuName || 0 == _tcsicmp(menuName, thisMenuName.c_str()))
 			{
-				size_t subMenuItems = (size_t)GetMenuItemCount(mii.hSubMenu);
+				int subMenuItems = GetMenuItemCount(mii.hSubMenu);
 				for (idx_t subMenuPos = 0; subMenuPos < subMenuItems; ++subMenuPos)

```

Comment:
As GetMenuItemCount() could return -1 in case of an error the for loop with unsigned idx_t could create a problem.